### ISSUE TITLE
[Support Admin] Incorrect scope attribute on Pending Organization table

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5827]
## Area or Component

Support Adming > Edit or verify organization identity

## Quick Summary

The incorrect scope is being used on column headers.

## Accessibility Violation

[Incorrect scope](https://cdc-devtools.dequecloud.com/issues/43c6cd10-c931-40ec-9583-24e2efe8fbf1)

## Action Requested

Change the scope to `col` instead of `row` on `Administrator`, `Contact Information`, and `Created`

## Screenshots (optional)

![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/799afae6-e408-44a9-a529-200e5aa780fe)
